test/unit/storage/utils/field.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable mocha/no-pending-tests */
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const rewire = require('rewire');
+
+const Field = rewire('../../../../storage/utils/field');
```

Comment:
also, here prefer not to use `rewire`, I think it would be better to just use stub